lib/asar.js

```diff
@@ -199,6 +203,9 @@ module.exports.extractAll = function (archive, dest) {
       // it's a file, extract it
       const content = disk.readFileSync(filesystem, filename, file)
       fs.writeFileSync(destFilename, content)
+      if (file.executable) {
+        fs.chmodSync(destFilename, '755')
+      }
```

Comment:
this seems unrelated to the rest of the PR?

Reply:
Yeah, was just a bug that happened to be fixed as part of running the tests 🤷